src/test/java/operation/handler/test/OperationHandlerTest.java

```diff
@@ -0,0 +1,49 @@
+package operation.handler.test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import core.basesyntax.basesyntax.db.Storage;
+import core.basesyntax.basesyntax.model.FruitTransaction;
+import core.basesyntax.basesyntax.strategy.OperationHandler;
+import core.basesyntax.basesyntax.strategy.impl.ReturnHandlerImpl;
+import core.basesyntax.basesyntax.strategy.impl.SupplyHandlerImpl;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class OperationHandlerTest {
```

Comment:
1. We should test implementations, not interface
2. Please expand test cases to check different cases (e.g. that balance operation will create new fruit record in DB, purchase cannot go to negative balance etc.)